nss-day-cohort-48/the-worst-site-ever-eliminatorz
# Consolidated Images, Fixed a typo in shawn.html

## To Test
- `fetch --all` and `checkout vc-imagefolder`
- make sure all the images load
- make sure shawn.html links are properly styled
